test-cases/performance/ycsb/perf-base.yaml

```diff
@@ -0,0 +1,59 @@
+test_duration: 600
+
+pre_create_keyspace: [
+  "CREATE KEYSPACE ycsb WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': 3 };",
+  "CREATE TABLE ycsb.usertable (y_id varchar primary key, field0 varchar, field1 varchar, field2 varchar, field3 varchar, field4 varchar, field5 varchar, field6 varchar, field7 varchar, field8 varchar, field9 varchar);"
+]
+
+# there is no sense to use more than 1 loader for 3 nodes cluster,
+# but this is just for example
+prepare_write_cmd:
+  - >-2
+    -threads 84
+    -p readproportion=0 -p insertproportion=1
+    -p fieldcount=10 -p fieldlength=128
+    -p scylla.username=cassandra -p scylla.password=cassandra -p scylla.tokenaware=true
+    -p scylla.coreconnections=14 -p scylla.maxconnections=14
+    -p scylla.readconsistencylevel=QUORUM -p scylla.writeconsistencylevel=QUORUM
```

Comment:
Do we measure the prepare results as well?
If not, maybe worth to write with ALL to avoid cases where the prepare was laggy and it affects the results of the actual stress.
(We do it in other performance tests).

Reply:
I can change this, I was using this as default (in contrast to the ONE used for the measuring)

Reply:
In all perf tests we are saving the results of prepare in ES, but never show them in the reports

Reply:
Changed to  ALL